css/theme.css

```diff
@@ -49,18 +49,24 @@ body {
 }
 
 header {
+  width: auto;
+  max-width: 1000px;
+  padding: 200px;
+}
+
+.header-color-block {
+  border-radius: 50px;
   background: var(--yellow);
   display: block;
   height: 100px;
   width: 100px;
   margin-bottom: 40px;
 }
 
-#content {
+main {
   width: auto;
   max-width: 1000px;
-  margin: 100px;
-  padding: 100px;
+  padding: 200px;
```

Comment:
padding is interacting with something else that I'm not sure, causing a width overrun